src/js/d3_svg_cell_renderer.js

```diff
@@ -0,0 +1,207 @@
+var utils = require('./utils');
```

Comment:
was I using camel case for files?

or do you mean I'm using camel case elsewhere so why use underscore case for files? In which case my answer to why the difference is that files are different than code. I don't have a justification besides what feels nice to me - if it's consensus that this is weird then we can change that